graphql_api/types/test_results_aggregates/test_results_aggregates.py

```diff
@@ -0,0 +1,43 @@
+from ariadne import ObjectType
+
+test_results_aggregates_bindable = ObjectType("TestResultsAggregates")
+
+
+@test_results_aggregates_bindable.field("totalRunTime")
+def resolve_total_run_time(obj, _) -> float:
+    return obj["total_run_time"]
+
+
+@test_results_aggregates_bindable.field("slowestTestsRunTime")
+def resolve_slowest_tests_run_time(obj, _) -> float:
+    return obj["slowest_tests_duration"]
+
+
+@test_results_aggregates_bindable.field("totalFails")
+def resolve_total_fails(obj, _) -> int:
+    return obj["fails"]
+
+
+@test_results_aggregates_bindable.field("totalSkips")
+def resolve_total_skips(obj, _) -> int:
+    return obj["skips"]
+
+
+@test_results_aggregates_bindable.field("totalRunTimePercentChange")
+def resolve_run_time_percent_change(obj, _) -> float | None:
+    return obj.get("total_run_time_percent_change")
+
+
+@test_results_aggregates_bindable.field("slowestTestsRunTimePercentChange")
+def resolve_slowest_tests_run_time_percent_change(obj, _) -> float | None:
+    return obj.get("slowest_tests_duration_percent_change")
```

Comment:
nit: why is this called duration vs. runTime like the other? and to make resolver name